docs/common/craft-parts/reference/plugins/uv_plugin.rst

```diff
@@ -0,0 +1,100 @@
+.. _craft_parts_uv_plugin:
+
+uv plugin
+=========
+
+The uv plugin can be used for Python projects that use the uv build tool.
+
+Keywords
+--------
+
+A sequence of subsections specifying the keywords provided by the plugin and
+describe how the plugin uses them using the same format for keywords used in
+the :ref:`part_properties` reference.
+
+In addition to the common :ref:`plugin <part-properties-plugin>` and
+:ref:`sources <part-properties-sources>` keywords, this plugin provides the
+following plugin-specific keywords:
+
+uv-extras
+~~~~~~~~~
+**Type:** list of strings
+
+Extra dependency groups to build with. Each element of the list is passed
+exactly as ``--extra GROUP``.
+
+Environment variables
+---------------------
+
+Along with the variables defined by the :ref:`Python plugin <craft_parts_python_plugin-environment_variables>`,
+this plugin defines additional variables defined below.
+
+.. note::
+
+    This section describes how this plugin uses ``uv``-specific variables. For
+    more information, as well as a complete list of environment variables
+    for ``uv``, see the `uv documentation <https://docs.astral.sh/uv/configuration/environment/>`__.
```

Comment:
Small formatting convention that I'm trying to enforce:

- For software names, even if they share the name with their CLI command and start with a lowercase letter, inline code isn't needed.
- When you _are_ talking about the software's command, then dressing it with inline code is fine.

A convention that started before me was that we use two spaces for indentation in RST.

```suggestion
  This section describes how this plugin uses uv-specific environment variables. For
  more information, as well as a complete list of environment variables
  for uv, see the `uv documentation <https://docs.astral.sh/uv/configuration/environment/>`_.
```